### DISCUSSION TITLE
lower the power of the engines

### DISCUSSION THREAD

**Discussion by FF1970** [iNavFlight/inav#10403] [Q&A]
hello
i can't find like in betaflight the possibility to lower the power of the engines in percentage?
i have a 3 inch drone with an MTF 01 sensor
the throttle max function causes stability problems if i lower it
i would like to avoid playing on the radio with output ch3 max

thank

**Comment by Jetrell** [comment]
That would be `throttle_scale` 

But for altitude hold using the range finder. You may want to lower `nav_mc_auto_climb_rate` and `nav_mc_manual_climb_rate`

**Comment by sensei-hacker** [comment]
It's slightly unclear what you're actually wanting to accomplish because there's nothing you _could_ do in the radio that would limit the motor output.

Are you wanting to make the INPUT less sensitive so it's easier to fly, or are you worried about burning a motor and you want to limit the OUTPUT?

You could reduce the sensitivity of the throttle stick INPUT. That it would be same as if you just moved the stick less - it would no have no effect on the motor OUTPUT used for stabilization and maneuvering.

One thing you might experiment with is in the mixer you can reduce any weights below 100% (1). For example you could set each throttle weight 0.75 (75%). That would reduce the output of the throttle portion. 

If you want less output for pitch, roll, and yaw that's best done in the PId tuning page, starting with the "Rates and Expo" tab before changing anything in the "PID Gains" tab.

**Comment by FF1970** [comment]
hello
i want to be able to discover the drone easily with the mtf01 sensor
the flights will be indoor

that's why i want to limit the throttle power to 85% max to restrict the max throttle on the gas stick

today i do it on the radio in output ch3 (-100% + 40%) and it works but i would only like to do it in inav 7.1.2
